tests/TestHelpers/GraphHelper.php

```diff
@@ -1019,4 +1019,58 @@ public static function getApplications(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $groupId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getUsersWithFilterMemberOf(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		string $groupId
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'users' . '?$filter=memberOf/any(m:m/id ' . "eq '$groupId')");
+		return HttpRequestHelper::get(
+			$url,
+			$xRequestId,
+			$user,
+			$password,
+			self::getRequestHeaders()
+		);
+	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param array $groupIdArray
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getUsersOfSeveralGroups(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		array $groupIdArray
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'users' . '?$filter=memberOf/any(m:m/id ' . "eq '$groupIdArray[0]') " . "and memberOf/any(m:m/id eq '$groupIdArray[1]')");
```

Comment:
i doubt if we want to get users of more than three groups, this `url` would not work right? how about making it more dynamic if possible.

Reply:
Yes, that is correct. But I am not sure that we will add tests in which we test more than three groups. I don't see the need for such tests. That's why I didn't make it complicated. 
I can change method name like `getUsersOfTwoGroups`

Reply:
sure !!